torch/testing/_internal/common_device_type.py

```diff
@@ -1339,8 +1339,13 @@ def skipCUDAIfNoMagmaAndNoLinalgsolver(fn):
         return skipCUDAIfNoMagma(fn)
 
 # Skips a test on CUDA when using ROCm.
-def skipCUDAIfRocm(fn):
-    return skipCUDAIf(TEST_WITH_ROCM, "test doesn't currently work on the ROCm stack")(fn)
+def skipCUDAIfRocm(func=None, msg="test doesn't currently work on the ROCm stack"):
```

Comment:
Make msg a kwarg only arg to make sure it is no interpreted as a fn.
```suggestion
def skipCUDAIfRocm(func=None, *, msg="test doesn't currently work on the ROCm stack"):
```

Reply:
Thank you @albanD, this makes more sense. Will update the PR with this change.